Tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Command;
+
+use Sonata\MediaBundle\Command\RemoveThumbsCommand;
+use Sonata\MediaBundle\Model\MediaManagerInterface;
+use Sonata\MediaBundle\Provider\Pool;
+use Sonata\MediaBundle\Tests\Entity\Media;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @author Anton Dyshkant <vyshkant@gmail.com>
+ *
+ * @requires function Symfony\Component\Filesystem\Tests\FilesystemTestCase::setUpBeforeClass
+ * @requires function Symfony\Component\Console\Input::setStream
+ */
+class RemoveThumbsCommandTest extends TestCase
+{
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @var Application
+     */
+    protected $application;
+
+    /**
+     * @var RemoveThumbsCommand
+     */
+    protected $command;
+
+    /**
+     * @var CommandTester
+     */
+    protected $tester;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Pool
+     */
+    private $pool;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|MediaManagerInterface
+     */
+    private $mediaManager;
```

Comment:
Why are some protected?

Reply:
I made this test case on the `CleanMediaCommandTest` sample. In `CleanMediaCommandTest` class  some properties are `protected`. I don't need this. Should I change they to `private`?

Reply:
Make everything private by default. http://fabien.potencier.org/pragmatism-over-theory-protected-vs-private.html